src/Microsoft.Data/Microsoft.DataFrames.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Generally, the project name and the namespace match. Should the namespace be `Microsoft.DataFrames` instead of `Microsoft.Data` or should the project file be changed to `Microsoft.Data.csproj`.

Reply:
I changed it back to `Microsoft.Data` for now. I'll come back to this in the next patch and change the assembly and namespace names. I have other patches built on top of this are getting harder and harder to rebase with these .csproj changes.

Reply:
I am going to push-back **slightly** on renaming the assembly after the initial check-in since it would end up publishing a NuGet package to our feed (with the same name) which would become out-dated and have to be cleaned up (after the rename). That said, **it's generally fine if we do that,** but if we know that a rename is required up front, it might be better to just settle on the name now.

Although, I understand your rebase/patch work that is being done on top of this PR, so I don't have any significant pushback form keeping it as is and renaming it later :)

Reply:
Hmmm, if I name the assembly Microsoft.DataFrame, the type is also called DataFrame, so usage now needs to be qualified as DataFrame.DataFrame, which seems silly. Microsoft.DataFrames is misleading considering there is likely only going to be 1 DataFrame. What about Microsoft.Data.DataFrame with a namespace of Microsoft.Data?